modelling-jakarta/src/main/resources/perClassConstants.properties

```diff
@@ -0,0 +1 @@
+org/axonframework/modelling/saga/AbstractResourceInjector.class=notransform.properties
```

Comment:
Curious: I expect you need to add the `AbstractResourceInjector` to be ignored for the `DEFAULT_INJECT_ANNOTATIONS` constant that contains `javax.inject.Inject`, right?

Reply:
Or is that already solved by the `notransform.properties` file? :thinking:

Reply:
Correct. By default, it uses the default Jakarta rules (see `<jakartaDefaults>true</jakartaDefaults>` configured in `pluginManagement` in parent POM). For `AbstractResourceInjector`, we need a custom rule (that is what `notransform.properties` is) on top of that because the class needs to be aware of both namespaces.

Reply:
Awesome, thanks for clarifying that, @azzazzel!